lib/mix/lib/mix/compilers/erlang.ex

```diff
@@ -313,7 +313,9 @@ defmodule Mix.Compilers.Erlang do
     for {_, warnings} <- entries,
         {file, issues} <- warnings,
         {location, module, message} <- issues do
-      IO.puts("#{file}:#{location_to_string(location)} warning: #{module.format_error(message)}")
+      message = "#{file}:#{location_to_string(location)} warning: #{module.format_error(message)}"
+
+      IO.puts(:stderr, IO.ANSI.format([:yellow, :bright, message]))
```

Comment:
The Elixir compiler does not wrap the the whole message in yellow, only the warning part, not in bright, and it ultimately uses another format, so I am not sure we should go for consistency here. We should probably be more consistent with the Erlang compiler warnings and error messages instead.

Reply:
Those are different warnings emitted by different places, and different on purpose, since they have implications. I would prefer to stay consistent within each library: Erlang, Mix, and Elixir.

Reply:
So what is the desired format for warnings in the Erlang and Mix library?

Reply:
Probably the formats that we have right now? :) although I will double check if the Erlang one is still relevant.